static/bidder-info/compass.yaml

```diff
@@ -0,0 +1,18 @@
+maintainer:
+  email: "sa-support@brightcom.com"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+    
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://sa-cs.deliverimp.com/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
```

Comment:
I tried to test the redirect URL and got an 'Unable to connect' error:
```
An error occurred during a connection to sa-cs.deliverimp.com. 
```
Are we positive the URL is correct?

Reply:
Yes, we are sure the URL is correct. We are currently configuring the server.